springboard/templates/base.jinja2

```diff
@@ -14,14 +14,13 @@
     <div id="header">
         <div class="lang">
             <div>
-                {% for code, name in languages %}
-                <span>
-                    <a href="/locale/?language={{code}}">{{name}}</a>
-                    {% if not forloop.last %}
-                        <span> |</span>
-                    {% endif %}
-                </span>
+                {% for lang_code, lang_name in view.get_display_languages() %}
```

Comment:
Instead of calling `view.get_display_languages` here, perhaps add it to the view context (there's a `context` method on the view class).